test/presenters/topic_presenter_test.rb

```diff
@@ -29,6 +29,15 @@ class TopicPresenterTest < ActiveSupport::TestCase
   test "returns accordion content data" do
     accordion_content = presented_topic.accordion_content
     first_accordion_section = {
+      data_attributes: {
```

Comment:
Is this needed?

Reply:
Yeah, sadly - the whole hash is tested and expected (hard coded)